CONTRIBUTING.md

```diff
@@ -0,0 +1,119 @@
+## Development of web-ext
+
+To get started, first install it from source as documented in [readme](https://github.com/mozilla/web-ext/blob/master/README.md).
```

Comment:
Because `CONTRIBUTING.md` most like appears when filing an issue or creating a PR, this shouldn't be needed. In either of those situations, the developer has already obtained the source (or didn't need it).